src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,47 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
-        List<Product> products = getAllProductsFromShoppingCart(userId);
+        logger.info("completeOrder method was called for userId: {}", userId);
+        List<Product> products = null;
+        try {
+            products = getAllProductsFromShoppingCart(userId);
+            logger.debug("Products retrieved from shopping cart for userId: {}: {}",
+                    userId, products);
+        } catch (Exception e) {
+            logger.error("Failed to retrieve products from shopping cart for userId: {}",
+                    userId, e);
+        }
+
         Order order = new Order(products, userId);
+        logger.info("Order created for userId: {} with products: {}", userId, products);
         // NOTE: In production ready code this order identifier should be generated by DB
         // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
+        logger.debug("Order ID set to 1 for userId: {}", userId);
+
+        logger.info("Order successfully completed for userId: {}", userId);
```

Comment:
The log message indicating that the order is completed is good, but it would be more informative if it included the order ID as well.